### PR TITLE
👩 ‍ General improvements

### LINKED ISSUE #312: Change status badge on Readme from CircleCI to GitHub Actions
Replace with https://img.shields.io/github/checks-status/hack4impact-uiuc/ymca/master?style=flat-square

### PR DESCRIPTION
**Pull request by n3a9**
- Fix all linting errors
- Fix README badge (Closes #312)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 </h1>
 
 <p align="center">
-    <img src="https://img.shields.io/circleci/build/github/hack4impact-uiuc/ymca/master?style=flat-square">
+    <img src="https://img.shields.io/github/checks-status/hack4impact-uiuc/ymca/master?style=flat-square">
```
Comment (Leundai) [inline README.md:9]: Very nice

**Review by jackieo5023 — COMMENTED** [review]
image
image
looks like adding the button made these look a bit... strange

**Comment by arpanlaha** [comment]
> image
> image
> looks like adding the button made these look a bit... strange

yeah you'll need to remove the button css

**Review by arpanlaha — APPROVED** [review]
🍪

**Pull request merged**